src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -516,7 +516,15 @@ private string UFormatDateString(DateTime dateTime)
                             break;
 
                         case 'u':
-                            sb.Append((int)dateTime.DayOfWeek);
+                            if (dateTime.DayOfWeek == DayOfWeek.Sunday)
+                            {
+                                sb.Append(7);
+                            }
+                            else
+                            {
+                                sb.Append((int)dateTime.DayOfWeek);
+                            }
+                            
```

Comment:
I suggest more short and readable:
```suggestion
                            var dayOfWeek = dateTime.DayOfWeek == DayOfWeek.Sunday ? 7 : (int)dateTime.DayOfWeek);
                            sb.Append(dayOfWeek );

```

Reply:
How about a little more point free, even?

```csharp
sb.Append( dateTime.DayOfWeek == DayOfWeek.Sunday ? 7 : (int)dateTime.DayOfWeek );
```

Reply:
Alternatively, I could simply use a format.

```csharp
sb.AppendFormat("{0:0;0;\7}", (int)dateTime.DayOfWeek);
```

Reply:
I prefer my suggestion as more readable.